style.css

```diff
@@ -101,6 +101,44 @@ p {
   font-weight: 5px;
 }
 
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] The hamburger menu should not be displayed on the desktop view and the nav links should be included in the top bar, kindly implement these changes so your design looks like the Figma design
- [ ] Please consider removing the underline for the logo, so it looks like Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220611_091727](https://user-images.githubusercontent.com/25227022/173180464-68194886-5f8e-4527-83eb-dff2d13a9bdd.png)|![Screenshot_20220611_093228](https://user-images.githubusercontent.com/25227022/173180473-cb41dd38-e80e-4d57-9576-07dc02400162.png)|